src/style.css

```
@@ -0,0 +1,124 @@
+body,
+h3,
+p {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__card {
+  margin: 0 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+}
+
+.image {
+  margin-top: 32px;
+  object-fit: contain;
+  display: flex;
+  height: 134px;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+
+  margin-top: 40px;
+}
+
+.productCode {
```

```suggestion
.card__product_code {
```
we dont use CamelCase in class names